src/node/genesis_gen.h

```diff
@@ -148,6 +131,29 @@ namespace ccf
       return id;
     }
 
+    void activate_member(MemberId member_id)
+    {
+      auto members = tx.get_view(tables.members);
+      auto member = members->get(member_id);
+      if (!member.has_value())
+      {
+        throw std::logic_error(fmt::format(
+          "Member {} cannot be activated as they do not exist", member_id));
+      }
+
+      if (member->status == MemberStatus::ACCEPTED)
+      {
+        member->status = MemberStatus::ACTIVE;
+        if (get_active_members_count() >= max_active_members_count)
+        {
+          throw std::logic_error(fmt::format(
```

Comment:
consider using cc_logic_error, this will also produce a stack trace when running in virtual

Reply:
@ashamis should we just do a pass and replace'em all?

Reply:
Raised https://github.com/microsoft/CCF/issues/1324 as it is something we should do consistently in our codebase.